hack/quick-install.sh

```diff
@@ -32,19 +32,9 @@ EOF
 bash cmd/operator/generate.sh
 helm template kubearchive charts/kubearchive -n kubearchive \
     --include-crds \
-    --set "global.production=true" \
-    --set "database.enabled=false" > /tmp/kubearchive-not-resolved.yaml
+    --set "global.production=true"  > /tmp/kubearchive-not-resolved.yaml
 ko resolve -f /tmp/kubearchive-not-resolved.yaml --base-import-paths > /tmp/kubearchive.yaml
 kubectl apply -n kubearchive -f /tmp/kubearchive.yaml
 
-cat <<EOF > /tmp/patch.yaml
-stringData:
-  POSTGRES_URL: postgresql.databases.svc.cluster.local
-  POSTGRES_USER: kubearchive
-  POSTGRES_DB: kubearchive
-  POSTGRES_PASSWORD: 'P0stgr3sdbP@ssword'  # notsecret
-EOF
-kubectl patch -n kubearchive secrets kubearchive-database-credentials --patch-file /tmp/patch.yaml
```

Comment:
I think we need to keep this. We have #380 to review how we provide the secret, but the state right know is that the user needs to `patch` and `rollout restart` after installing.

Reply:
I removed it because the secret already had these values. 
If I'm understanding this correctly I should put empty values in the secret based on a chart value (like we have before for `database.enabled`) so that this patch actually do something, right?

Reply:
I don't think you need to put empty values based on a chart value, just put empty values. The `patch` will configure it properly.

Reply:
Then, the usage for "development" won't work.
The instructions we currently have for development are:
1. Run `hack/install-database.sh`
2. Install the helm chart

The patch is being done in the `hack/quick-install.sh` meaning, it won't be done in the development as we have it.

Other alternative is to add one more step to the development procedure to run the patch after installing the chart.

Reply:
Ah, right. Then maybe recovering the `_helpers.tpl` block and using `.Values.global.production` as conditional may work.

Reply:
I put it directly on the secret, looks good to you? or you still prefer to recover the `_helpers.tpl` block?

Reply:
Let's keep it on the secret and see where #380 lead us.

Reply:
See my other comment.  Just put all the development values into the secret. I don't think you'll need this patch at all then.